__tests__/app.test.js

```diff
@@ -246,6 +246,84 @@ describe("my express project", () => {
           expect(articles).toBeSortedBy("created_at", { descending: true });
         });
     });
+
+    it("200 sort_by, which sorts the articles by any valid column (defaults to date)", () => {
+      return request(app)
+        .get("/api/articles?sort_by=votes")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy("votes", { descending: true });
+        });
+    });
+
+    it("200 order, which sorts the articles by any valid column (defaults to date)order(defaults to descending)", () => {
+      return request(app)
+        .get("/api/articles?sort_by=title&order=asc")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy("title");
+        });
+    });
+
+    it("400 sort_by, responds with bad request if sort column is invalid)", () => {
+      return request(app)
+        .get("/api/articles?sort_by=comment_id&order=desc")
+        .expect(400)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("invalid sort request");
+        });
+    });
+
+    it("400 order, responds with bad request if order is invalid)", () => {
+      return request(app)
+        .get("/api/articles?sort_by=title&order=high")
+        .expect(400)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("invalid order request");
+        });
+    });
+
+    it("400 sort_by, responds with bad request if sort column and sort order is invalid", () => {
+      return request(app)
+        .get("/api/articles?sort_by=comment_id&order=higher")
+        .expect(400)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("invalid sort request");
+        });
+    });
+
+    it("200 filter by topic, responds with articles with the filter topic", () => {
+      return request(app)
+        .get("/api/articles")
+        .query({ topic: "mitch" })
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toHaveLength(11);
+          expect(articles).toBeInstanceOf(Array);
+        });
+    });
+
```

Comment:
would be a good idea to add another assertion here to check articles have the correct keys